### PR TITLE
CIV-14062 action caption on APPLICATION_ADD_PAYMENT status

### PR DESCRIPTION
**Pull request by david-jeronimo**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-14062

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by manish14836 — DISMISSED** [review]

File: src/main/modules/i18n/locales/en.json [hunk 1]
```diff
@@ -2794,7 +2794,7 @@
         "STATES": {
           "PENDING_APPLICATION_ISSUED": "General Application Issue Pending",
           "AWAITING_RESPONDENT_RESPONSE": "Awaiting Respondent Response",
-          "APPLICATION_ADD_PAYMENT": "Application Additional Payment",
+          "APPLICATION_ADD_PAYMENT": "Pay the additional application fee",
```
Comment (manish14836) [inline src/main/modules/i18n/locales/en.json:2797]: Is this change required in cy.json as well?

**Review by paul-pearson — APPROVED** [review]

File: src/main/modules/i18n/locales/en.json (hunk 1 above)
Comment (david-jeronimo) [reply src/main/modules/i18n/locales/en.json:2797]: All these fields are still in English in cy.json

Comment (manish14836) [reply src/main/modules/i18n/locales/en.json:2797]: I think both files should be consistent.

Comment (david-jeronimo) [reply src/main/modules/i18n/locales/en.json:2797]: You are right. Both changed now. Thanks

**Review by manish14836 — APPROVED** [review]
